modules/tinymce/CHANGELOG.md

```diff
@@ -49,6 +49,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - A newline could not be inserted when the selection was restored from a bookmark after an inline element with a `contenteditable="false"` attribute. #TINY-9194
 - The global `tinymce.dom.styleSheetLoader` was not affected by the `content_css_cors` option. #TINY-6037
 - The caret was moved to the previous line when a text pattern executed a `mceInsertContent` command on Enter key when running on Firefox. #TINY-9193
+- The toolbars are not rendered during the dragging of CEF elements. #TINY-9305
```

Comment:
- [ ] I can't find using the CEF abbreviation in the changelog. Consider such a wording:

Quick toolbars were incorrectly rendered during the dragging of `contenteditable="false"` elements

- [ ] And also move the changelog entry to the Unreleased Fixed section since 6.3 is closed up

Reply:
ok, changed

Reply:
moved, sorry I miss read that part